src/wmnds/components/content-card/_content-card.scss

```diff
@@ -0,0 +1,71 @@
+.wmnds-content-card {
+  border-bottom: 5px solid get-color(primary);
+  background-color: $white;
+
+  @at-root a#{&} {
+    display: block;
+    color: get-color(text);
+    text-decoration: none;
+  }
+
+  // Modifiers
+  &--yellow {
+    background-color: get-color(plannedDisruption);
+  }
+
+  // Elements
+  &__title {
+    margin-bottom: $size-sm;
+  }
+
+  &__list {
+    padding: 0;
+    list-style-type: none;
+
+    &-item {
+      margin: $size-sm 0 0;
+
+      .wmnds-btn {
+        width: 100%;
+        text-align: left;
+      }
+    }
+  }
+
+  &__image {
+    width: 100%;
+    object-fit: cover;
+  }
+
+  &__disruption-grid {
+    display: flex;
+    justify-content: space-between;
+    flex-flow: row wrap;
+
+    & > &-item {
+      display: flex;
+      width: calc(50% - (#{$size-sm} / 2));
+      flex-flow: row wrap;
+
+      > * {
+        margin: 0 auto $size-sm;
+      }
+
+      .wmnds-disruption-indicator-medium {
+        display: flex;
+        width: 100%;
+        justify-content: space-between;
+      }
```

Comment:
This is a good solution but it is a duplicate of what has been provided as [utility classes](https://deploy-preview-387--wmnetwork.netlify.app/styles/utility-classes/).

Can you remove this styling/classes and update to the utility classes.

**Tip for future**
Utility classes should be used on patterns but direct styling should be used on components.
For example don't use the margin/padding util classes on a component as the margin should be on the component not via a utility. But as a pattern is made up of multiple components etc. it is fine to use the utility classes to assist with layout etc.

Reply:
Yeah makes sense, will use the utilities 👍🏾